### PR TITLE
Run the code formatter on Logger

### PR DESCRIPTION
**Pull request by mhinz**

### CODE REVIEW

File: lib/logger/lib/logger.ex [hunk 1]
```diff
@@ -470,14 +472,14 @@ defmodule Logger do
...
   """
   @spec compare_levels(level, level) :: :lt | :eq | :gt
-  def compare_levels(level, level), do:
-    :eq
-  def compare_levels(left, right), do:
-    if(level_to_number(left) > level_to_number(right), do: :gt, else: :lt)
+  def compare_levels(level, level), do: :eq
+
+  def compare_levels(left, right),
+    do: if(level_to_number(left) > level_to_number(right), do: :gt, else: :lt)
```
Comment (mhinz) [inline lib/logger/lib/logger.ex]: Is this okay or shouldn't it be inline in the first place?

```elixir
def compare_levels(left, right) do
  if(level_to_number(left) > level_to_number(right), do: :gt, else: :lt)
end
```

Comment (whatyouhide) [reply lib/logger/lib/logger.ex]: Let's drop the `do:` syntax IMO and have

```elixir
def compare_levels(level, level) do
  :eq
end

def compare_levels(left, right) do
  if level_to_number(left) > level_to_number(right), do: :gt, else: :lt
end
```

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Comment by yordis** [comment]
@josevalim should I add those parenthesis in the typespec? Right now I am not putting them but I seems I should be doing it I guess

**Pull request merged**